icons/solidjs/solidjs-original-wordmark.svg

```diff
@@ -0,0 +1 @@
4.4" y1="74.5" y2="260.8" gradientTransform="translate(-.394 50.573) scale(.17312)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#4377bb"/><stop offset=".5" stop-color="#1a336b"/><stop offset="1" stop-color="#1a336b"/></linearGradient></defs><path fill="#fff" d="M40.121 67.517a4.374 4.374 0 0 0 2.952 1.24c1.24 0 2.26-.73 2.26-1.969 0-2.843-5.65-2.15-5.65-5.65 0-1.64 1.458-3.024 3.535-3.024a4.374 4.374 0 0 1 3.026 1.093l-.55.948a3.783 3.783 0 0 0-2.48-.984c-1.457 0-2.368.947-2.405 1.931 0 2.697 5.65 1.932 5.65 5.65 0 1.676-1.313 3.061-3.427 3.061a4.87 4.87 0 0 1-3.57-1.458zm17.895-9.33a5.733 5.733 0 0 1 5.76 5.795 5.832 5.832 0 0 1-5.76 5.904h-.036a5.832 5.832 0 0 1-5.76-5.904v-.037a5.733 5.733 0 0 1 5.796-5.758zm-.036 10.68a4.738 4.738 0 0 0 4.628-4.845v-.04c0-2.697-2.04-4.775-4.592-4.775a4.636 4.636 0 0 0-4.629 4.738c0 2.77 2.041 4.884 4.593 4.92zm12.355-10.46h1.094l-.038 10.313h5.25v.984h-6.38zm12.356.035h1.094l-.037 11.335h-1.093zm8.456 0h3.645c3.39 0 5.685 2.077 5.685 5.686 0 3.608-2.296 5.65-5.685 5.65h-3.645zm3.5 10.35c2.77 0 4.628-1.64 4.664-4.664.037-3.026-1.859-4.666-4.629-4.666h-2.44l-.037 9.33zm15.37.985a3.743 3.743 0 0 0 2.572-.919 3.28 3.28 0 0 0 1.042-2.587v-7.694h-2.278v7.526a1.53 1.53 0 0 1-.364 1.093 1.316 1.316 0 0 1-1.01.39q-1.07 0-1.563-1.257l-1.928 1.075q.944 2.373 3.528 2.373zm13.652.025c1.265 0 2.256-.313 2.967-.944a3.138 3.138 0 0 0 1.064-2.474 2.584 2.584 0 0 0-.91-2.067c-.613-.525-1.594-.977-2.942-1.363a4.286 4.286 0 0 1-1.37-.572.937.937 0 0 1-.402-.784.984.984 0 0 1 .401-.82 1.677 1.677 0 0 1 1.06-.31 2.067 2.067 0 0 1 2.082 1.356l1.884-1.03a3.663 3.663 0 0 0-1.483-1.766 4.436 4.436 0 0 0-2.38-.623 4.05 4.05 0 0 0-2.715.948 2.956 2.956 0 0 0-1.127 2.387 2.734 2.734 0 0 0 .846 2.045q.758.765 2.686 1.345a6.018 6.018 0 0 1 1.648.663.85.85 0 0 1 .433.762.999.999 0 0 1-.47.842 2.056 2.056 0 0 1-1.203.332 2.551 2.551 0 0 1-2.405-1.542l-1.968 1.017a3.955 3.955 0 0 0 1.636 1.906 5.008 5.008 0 0 0 2.668.692z"/><path fill="#76b3e1" d="M27.798 56.627s-9.166-6.745-16.258-5.189l-.519.173c-1.037.346-1.902.865-2.421 1.557l-.346.519-2.594 4.497 4.496.864c1.903 1.211 4.324 1.73 6.573 1.211l7.956 1.557z"/><path fill="url(#a)" d="M27.798 56.627s-9.166-6.745-16.258-5.189l-.519.173c-1.037.346-1.902.865-2.421 1.557l-.346.519-2.594 4.497 4.496.864c1.903 1.211 4.324 1.73 6.573 1.211l7.956 1.557z" opacity=".3"/><path fill="#518ac8" d="m8.6 56.627-.692.173c-2.94.865-3.805 3.632-2.248 6.054 1.73 2.248 5.361 3.459 8.302 2.594l10.723-3.632S15.518 55.07 8.6 56.627z"/><path fill="url(#b)" d="m8.6 56.627-.692.173c-2.94.865-3.805 3.632-2.248 6.054 1.73 2.248 5.361 3.459 8.302 2.594l10.723-3.632S15.518 55.07 8.6 56.627z" opacity=".3"/><path fill="url(#c)" d="M22.783 64.41a7.783 7.783 0 0 0-8.303-2.594L3.757 65.275l-3.46 6.054 19.372 3.286 3.46-6.227c.691-1.21.518-2.594-.346-3.978z"/><path fill="url(#d)" d="M19.323 70.464a7.783 7.783 0 0 0-8.302-2.595L.298 71.33s9.167 6.918 16.258 5.188l.519-.173c2.94-.864 3.978-3.632 2.248-5.88z"/></svg>
```

Comment:
This should be replaced with the version from their media assets page: https://www.solidjs.com/media